tools/nl/embeddings/build_embeddings.py

```diff
@@ -38,6 +43,8 @@
                     'Existing finetuned model folder name on GCS')
 flags.DEFINE_string('existing_model_path', '',
                     'Path to an existing model (local)')
+flags.DEFINE_string('vertex_ai_prediction_endpoint_id', '',
+                    'The ID of vertext AI prediction endpoint')
```

Comment:
Maybe add:  Not set for local SentenceTransformer based index, must be set for API-based index.